modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -9,81 +9,88 @@ Create a vSphere Windows virtual machine (VM) golden image.
 
 .Prerequisites
 
-* You have installed a cluster on vSphere configured with hybrid networking using OVN-Kubernetes.
-* You have defined a custom VXLAN port in your hybrid networking configuration to work around the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere[pod-to-pod connectivity issue between hosts].
+* You have created a private/public key pair, which is used to configure key-based authentication in the OpenSSH server.
+* You have installed link:https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell[Microsoft PowerShell]. PowerShell commands in this guide are distinguished by the `PS C:\>` prefix.
```

Comment:
> You have installed Microsoft PowerShell

I don't feel we need to call this out, PowerShell is installed by default and ready to use in Windows Server SAC: Windows Server 2004

> PowerShell commands in this guide are distinguished by the `PS C:\>` prefix.

Is a good note; but not a pre-req.

cc: @openshift/openshift-team-windows-containers

Reply:
I will remove the prereq. I will create the prefix portion as an independent note.